src/aiida_quantumespresso/parsers/base.py

```diff
@@ -25,39 +88,127 @@ def emit_logs(self, logging_dictionaries, ignore=None):
                 'error': ['Self-consistency was not achieved']
             }
 
-        :param logging_dictionaries: log dictionaries
+        :param logs: log dictionaries
         :param ignore: list of log messages to ignore
         """
         ignore = ignore or []
 
-        if not isinstance(logging_dictionaries, (list, tuple)):
-            logging_dictionaries = [logging_dictionaries]
+        if not isinstance(logs, (list, tuple)):
+            logs = [logs]
 
-        for logs in logging_dictionaries:
+        for logs in logs:
             for level, messages in logs.items():
                 for message in messages:
 
-                    if message is None:
-                        continue
-
                     stripped = message.strip()
 
-                    if not stripped or stripped in ignore:
+                    if stripped in ignore:
                         continue
 
-                    try:
```

Comment:
Oh, almost forgot: this one should also only really happen if we messed up the logs somehow, right?

Reply:
I may have added this because the `REPORT` level is an AiiDA only-thing. If the logging container were to be used outside of AiiDA where logging is not setup by it and so `Logger.report` wouldn't exist. But I guess it's fine to remove since this is anyway only usable by `aiida-quantumespresso` I think